src/cassandra_statement.rs

```diff
@@ -192,7 +189,12 @@ impl CassandraStatement {
             }
             "update" => CassandraStatement::Update(CassandraParser::parse_update(node, source)),
             "use" => CassandraStatement::Use(CassandraParser::parse_use(node, source)),
-            _ => CassandraStatement::Unknown(source.to_string()),
+            _ => CassandraStatement::Unknown(
```

Comment:
Isnt it more correct to include the entire unparseable source here? i.e. the previous implementation
When the user has a CassandraStatement::Unknown they want to be able to access the entire unparsed query, they dont particularly care how far the parser got before failing to parse the query.